Directory.Build.props

```diff
@@ -7,11 +7,36 @@
         <Version>2.1.0</Version>
     </PropertyGroup>
 
-    <PropertyGroup>
+    <PropertyGroup Condition="'$(Configuration)' == 'Release'">
```

Comment:
I wouldn't tie this to the configuration

Reply:
In that case I'd like to move the `<Version>` property to this block as well